app/controllers/users/new.js

```diff
@@ -25,12 +25,16 @@ export default Ember.Controller.extend({
     this.set("preferredPhone", "");
   },
 
+  formatMobileNumber() {
+    const mobile = this.get("mobilePhone");
+    return mobile.length
+      ? config.APP.HK_COUNTRY_CODE + this.get("mobilePhone")
+      : mobile;
+  },
+
```

Comment:
I'm not really seeing the difference with the previous version 🤔

Reply:
Previous code was returning 0, in this case, it will return blank.